test/parallel/test-sync-io-option.js

```diff
@@ -20,7 +20,7 @@ if (process.argv[2] === 'child') {
     execFile(process.execPath, args, function(err, stdout, stderr) {
       assert.equal(err, null);
       assert.equal(stdout, '');
-      if (/^WARNING[\s\S]*fs\.readFileSync/.test(stderr))
+      if (/^[\s\S]*fs\.readFileSync/.test(stderr))
```

Comment:
I'd suggest you just remove the `^` here, so the regex is `/WARNING[\s\S]*fs\.readFileSync/`.